lib/EnsEMBL/REST/Controller/Phenotype.pm

```diff
@@ -80,6 +80,45 @@ sub term: Chained('/') PathPart('phenotype/term') Args(2) ActionClass('REST') {
 }
 
 
+
+=pod
+
+/phenotype/region/Homo_sapiens/X:1000000-2000000?feature_type=Variation;
+
+feature_type = The type of feature associated with phenotype to retrieve (Variation/StructuralVariation/Gene/QTL).
+only_phenotypes = Only returns associated phenotype description and mapped ontology accessions for a lighter output.
+
+application/json
+text/x-gff3
+
+=cut
+
+has 'max_slice_length' => ( isa => 'Num', is => 'ro', default => 1e7);
+
+sub region_GET {}
+
+sub region: Chained('/') PathPart('phenotype/region') Args(2) ActionClass('REST') {
```

Comment:
Putting all the logic in sub region() guarantees rewrites if you want to add POST support later. It's too late for this code though.